src/RageUtil/Graphics/RageSurface_Save_PNG.cpp

```diff
@@ -1,6 +1,6 @@
 #include "Etterna/Globals/global.h"
 #include "RageUtil/File/RageFile.h"
-#include "RageUtil/Misc/RageLog.h"
+#include "Core/Services/Locator.hpp"
```

Comment:
This include can actually be removed instead.

Reply:
Fixed.